api/db-cdr/generate-cdr/libproject/genomic_manifests.rb

```diff
@@ -500,15 +501,24 @@ def _get_pooled_path(pathInfix, display_version_id, is_delta_release)
 end
 
 def build_copy_manifest_for_curation_section(input_section, ingest_bucket, dest_bucket, display_version_id)
-  path_prefix = "#{display_version_id}/#{input_section['destination']}"
-  ingest_base_path = File.join(ingest_bucket, path_prefix)
-  destination = File.join(dest_bucket, path_prefix)
+  # If an absolute path is provided, then use it.
+  if input_section['destination'].start_with?("gs://")
+    destination = input_section['destination']
+    path_prefix = destination.split(dest_bucket)[1]
```

Comment:
Do we need to account for the case where `destination` does not contain `dest_bucket` ?